data/migrations/20190405120711_actions.js

```diff
@@ -0,0 +1,17 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable("actions", actions => {
+    actions.increments();
+    actions.integer("project_id").unsigned();
+    actions
+      .foreign("project_id")
+      .references("id")
```

Comment:
Would like to see the foreign key be .notNullable(), also you should have a `.onDelete()` and `.onUpdate()` method chained to it as well.